orchagent/notifications.cpp

```diff
@@ -47,3 +50,18 @@ void on_switch_shutdown_request(sai_object_id_t switch_id)
 
     quick_exit(EXIT_FAILURE);
 }
+
+void on_switch_asic_sdk_health_event(sai_object_id_t switch_id,
```

Comment:
there are many comments in this file mentioned 
```
// don't use this event handler, because it runs by libsairedis in a separate thread
    // which causes concurrency access to the DB
```
do we really want to handle the notification here?

Reply:
Yes, we have to and we take steps to avoid conflict in db accessing.
See  [HLD](https://github.com/stephenxs/SONiC/pull/19/files#diff-95de87d9142acc745aa604e7d06b6e8e1eede3e78bffef95e2111acd7dbffdf5R645) for details